ml/src/test/scala/org/apache/spark/ml/spark/models/H2OAlgoTest.scala

```diff
@@ -58,6 +60,45 @@ class H2OAlgoTest extends FunSuite with SharedH2OTestContext {
     loadedModel.transform(dataset).count()
   }
 
+  test("H2OGLM with set modelId is trained mutliple times")
+  {
```

Comment:
NIT: any reason why `{` is on the new line ?

Reply:
There is no reason :-) I will fix it, thanks!